### PR TITLE
remove TESTRAIL_ENABLE and QTEST_ENABLE

### PR DESCRIPTION
**Pull request by ilya-varchenya**

### CODE REVIEW

File: src/com/qaprosoft/jenkins/pipeline/runner/maven/TestNG.groovy [hunk 1]
```diff
@@ -570,7 +569,7 @@ public class TestNG extends Runner {
                     clean()
                     customNotify()
 
-                    if (Configuration.get("testrail_enabled")?.toBoolean()) {
+                    if (Configuration.get("testrail_enabled")?.toBoolean() && !isParamEmpty(getCredentials(organization + "-testrail_service_url")) && !isParamEmpty(getCredentials(organization + "-testrail_creds"))) {
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/pipeline/runner/maven/TestNG.groovy:572]: no need organization on this level. Also let's do verification here in a little bit different way.
Next line might answer about registered credentials better:
String jobName = getCurrentFolderFullName(Configuration.TESTRAIL_UPDATER_JOBNAME)

if jobName is not empty - someone already registered testrail or qtest

File: src/com/qaprosoft/jenkins/pipeline/runner/maven/TestNG.groovy [hunk 2]
```diff
@@ -594,7 +593,7 @@ public class TestNG extends Runner {
                                     ]
                         }
                     }
-                    if(Configuration.get("qtest_enabled")?.toBoolean()){
+                    if (Configuration.get("qtest_enabled")?.toBoolean() && !isParamEmpty(getCredentials(organization+ "-qtest_service_url")) && !isParamEmpty(getCredentials(organization + "-qtest_token"))) {
                         String jobName = getCurrentFolderFullName(Configuration.QTEST_UPDATER_JOBNAME)
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/pipeline/runner/maven/TestNG.groovy:597]: the same change needed as in previous comment

File: src/com/qaprosoft/jenkins/pipeline/runner/maven/TestNG.groovy [hunk 3]
```diff
@@ -137,8 +137,7 @@ public class TestNG extends Runner {
 		setQTestCreds()
 
 		def ci_run_id = Configuration.get("ci_run_id")
-		Configuration.set("qtest_enabled", "true")
-		qTestUpdater.updateTestRun(ci_run_id)
+		qTestUpdater.updateTestRun(ci_run_id, organization)
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/pipeline/runner/maven/TestNG.groovy:140]: no need organization parameter

**Comment by okamara** [comment]
Tested on jenkins-master:5.1-beta3 , qps-infra - develop, pipeline- develop.
Open a job  configuration page and verify that there is no testrail_enable, and qtest_enable.
@ilya-varchenya is it correct verification case for this ticket?

**Pull request merged**